templates/taxi/car_detail.html

```diff
@@ -3,20 +3,31 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h1>
+    Drivers
+    {% if user in car.drivers.all %}
+      <a href="{% url 'taxi:car-delete-user' pk=car.id %}" class="btn btn-danger link-to-page">
+        Delete me from this car
+      </a>
+    {% else %}
+      <a href="{% url 'taxi:car-add-user' pk=car.id %}" class="btn btn-secondary link-to-page" style="background-color: green">
+        Assign me to this car
+      </a>
+    {% endif %}
```

Comment:
The template allows users to add or remove themselves as drivers from the car. Ensure that proper permissions are enforced in the views to prevent unauthorized users from making these changes. Additionally, consider handling cases where the user might not be logged in, which could cause errors when checking if the user is in `car.drivers.all`.